com.github.d4nj1.tlpui.yml

```diff
@@ -0,0 +1,34 @@
+app-id: com.github.d4nj1.tlpui
+command: tlpui
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+finish-args:
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission.

Reply:
Back in early 2021 there was no option to allow permissions on individual host files. Tlpui only needs read and write access to either `/var/run/host/etc/default/tlp` or `/var/run/host/etc/tlp.conf`depending on the TLP version.

Should the following work?
`- --filesystem=/var/run/host/etc/default/tlp:rw`
`- --filesystem=/var/run/host/etc/tlp.conf:rw`

Or is it still only folder paths like this then?
`- --filesystem=/var/run/host/etc:rw`

Reply:
None will. Even with `host`, you will have no write access to `/var/run/host/etc`.

Reply:
That being said, you could limit it to `host-etc`.

Reply:
Ok, then `host-etc` should be sufficient for me. The write to `/var/run/host/etc/default/tlp` or `/var/run/host/etc/tlp.conf`  happens anyways through a priviledged host copy-over-command from flatpak tmp-folder. At least that worked on my machine during development.

Reply:
There does not seem to exist a `host-usr`. How can this be achieved for `/usr` access?

Reply:
And also vor `/var` access?